vic/drivers/image/src/vic_init_output.c

```diff
@@ -193,10 +208,212 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining time dimenension in %s", nc->fname);
     }
 
+    // define the netcdf variable time
+    status = nc_def_var(nc->nc_id, "time", NC_DOUBLE, 1,
+                        &(nc->time_dimid), &(time_var_id));
+    if (status != NC_NOERR) {
+        log_err("Error defining time variable in %s", nc->fname);
+    }
+    status = nc_put_att_text(nc->nc_id, time_var_id, "standard_name",
+                             strlen("time"), "time");
+    if (status != NC_NOERR) {
+        log_err("Error adding attribute in %s", nc->fname);
+    }
+
+    // adding units attribute to time variable
+    if (global_param.time_units == TIME_UNITS_SECONDS) {
+        sprintf(strUnit, "seconds");
+    }
+    else if (global_param.time_units == TIME_UNITS_MINUTES) {
+        sprintf(strUnit, "minutes");
+    }
+    else if (global_param.time_units == TIME_UNITS_HOURS) {
+        sprintf(strUnit, "hours");
+    }
+    else if (global_param.time_units == TIME_UNITS_DAYS) {
+        sprintf(strUnit, "days");
+    }
+    else {
+        sprintf(strUnit, "-");
+    }
+
+    sprintf(str, "%s since 0001-01-01 00:00:00", strUnit);
+
+    status = nc_put_att_text(nc->nc_id, time_var_id, "units",
+                             strlen(str), str);
+    if (status != NC_NOERR) {
+        log_err("Error adding attribute in %s", nc->fname);
+    }
+
+    // adding calendar attribute to time variable
+    if (global_param.calendar == CALENDAR_STANDARD) {
+        sprintf(strCalendar, "standard");
+    }
+    else if (global_param.calendar == CALENDAR_GREGORIAN) {
+        sprintf(strCalendar, "gregorian");
+    }
+    else if (global_param.calendar == CALENDAR_PROLEPTIC_GREGORIAN) {
+        sprintf(strCalendar, "proleptic gregorian");
+    }
+    else if (global_param.calendar == CALENDAR_NOLEAP) {
+        sprintf(strCalendar, "noleap");
+    }
+    else if (global_param.calendar == CALENDAR_365_DAY) {
+        sprintf(strCalendar, "365 day");
+    }
+    else if (global_param.calendar == CALENDAR_360_DAY) {
+        sprintf(strCalendar, "360 day");
+    }
+    else if (global_param.calendar == CALENDAR_JULIAN) {
+        sprintf(strCalendar, "julian");
+    }
+    else if (global_param.calendar == CALENDAR_ALL_LEAP) {
+        sprintf(strCalendar, "all leap");
+    }
+    else if (global_param.calendar == CALENDAR_366_DAY) {
+        sprintf(strCalendar, "366 day");
+    }
+    else {
+        sprintf(strCalendar, "no calendar specified");
```

Comment:
raise an error here.  It is possible to not have a calendar attribute in the time variable but I don't think the current `implementation of `vic_time` can handle that.